app/models/journal.rb

```diff
@@ -54,6 +54,8 @@ class Journal < ApplicationRecord
   # Scopes to all journals excluding the initial journal - useful for change
   # logs like the history on issue#show
   scope :changing, -> { where(['version > 1']) }
+  scope :for_wiki_content, -> { where(journable_type: "WikiContent") }
```

Comment:
Sorry, nitpicking, but can you please add a blank line between these two and the one above. I was confused by the comment since it says "Scopes..."

Reply:
Sure, readibility is key. It's fixed now.
Thanks for spotting it.